### PR TITLE
Bart.BPLAT-12164.user id must always be string type

### PR DESCRIPTION
**Pull request by bartmcleod**
https://jira.aminocom.com/browse/BPLAT-12164

### CODE REVIEW

File: rest/snippets/UserConfiguration/UserProfile.yml [hunk 1]
```diff
@@ -10,10 +10,8 @@ UserProfile:
       description: A name for the profile to help the account-holder identify who it belongs to
     user_id:
       type: string
-      example: 5b56765e-fa7b-45d2-8dee-2869ec736cc9
-      description: >-
-        A system generated random user name.  The client should not POST this field when creating a profile.
-        The client will supply this username when calling the API.
+      example: 42
```
Comment (amino-garricklam) [inline rest/snippets/UserConfiguration/UserProfile.yml]: I think we will change the user id to UUID, the original example is correct.

Comment (bartmcleod) [reply rest/snippets/UserConfiguration/UserProfile.yml]: Why is it correct? It is not a uuid?

Comment (amino-garricklam) [reply rest/snippets/UserConfiguration/UserProfile.yml]: sorry, I mean `The client should not POST this field when creating a profile.` is correct

Comment (bartmcleod) [reply rest/snippets/UserConfiguration/UserProfile.yml]: The example should be representative of what we have in the data. Not be something totally random and suggestive of an implementation that we do not have (uuid).

We now just state our intention with the redundant user_id and it is also verified in the code that it matches the authenticated user

**Pull request merged**